private-templates-service/client/src/utils/getVersion/getVersion.ts

```diff
@@ -0,0 +1,8 @@
+import { Template } from 'adaptive-templating-service-typescript-node';
+
+export default function getVersion(template: Template): string {
```

Comment:
to add on to my previous comment about putting template utility function into utils, maybe we should make a file called `TemplateUtility.tsx` where we put all template related functions rather than having a file for each function.